apiserver/facades/client/highavailability/highavailability.go

```diff
@@ -170,6 +171,15 @@ func validateCurrentControllers(st *state.State, cfg controller.Config, machineI
 			// machines without any address are essentially not started yet
 			continue
 		}
+		if len(addresses) > 1 {
+			// ignore CIDR/32 et CIDR/64, which can be associated with a virtual IP (see https://bugs.launchpad.net/juju/+bug/2073986)
```

Comment:
```suggestion
			// ignore /32 and /64 addresses, which can be associated with a virtual IP (see https://bugs.launchpad.net/juju/+bug/2073986)
```

Reply:
Actually, this comment is wrong, should it be /128 address (for ipv6)?

Reply:
Done.